lib/absinthe/phase/document/arguments/data.ex

```diff
@@ -49,8 +49,8 @@ defmodule Absinthe.Phase.Document.Arguments.Data do
   def handle_node(%Input.Value{normalized: %Input.Object{fields: fields}} = node) do
     data =
       fields
-      |> Enum.filter_map(
-        &include_field?/1,
+      |> Enum.filter(&include_field?/1)
+      |> Enum.map(
         &{&1.schema_node.__reference__.identifier, &1.input_value.data}
```

Comment:
Hey, I think this may be best accomplished with a `for` comprehension:

```
data = for field <- fields, include_field?(field), into: %{} do
  {field.schema_node.__reference__.identifier, field.input_value.data}
end
```